lib/codeowners/cli/wizards/new_owner_wizard.rb

```diff
@@ -25,24 +28,55 @@ def suggest_fixing(line, new_owner)
 
         private
 
-        def prompt(line, new_owner)
-          ask(<<~QUESTION, limited_to: %w[y r i q])
-            Unknown owner: #{new_owner} for pattern: #{line.pattern}. Add owner to the OWNERS file?
-            (y) yes
-            (r) rename owner
-            (i) ignore owner in this session
-            (q) quit and save
+        def suggest_name_from_owners_list(new_owner)
+          require 'fuzzy_match'
+          search = FuzzyMatch.new(@owners_list.owners)
+          (suggested_owner, dice, _lev) = search.find_with_score(new_owner)
+          return suggested_owner if dice && dice > 0.6
+        end
+
+        def prompt(line, new_owner, suggested_owner)
+          prompt_options = build_prompt_options(suggested_owner)
+          accepted_options = filter_options(suggested_owner, prompt_options)
```

Comment:
Maybe we can improve this a bit because the `filter_options`  reject is not clear in the comparison of `c == 'o'`

We can introduce  static options: 
```ruby
DEFAULT_OPTIONS =  {
            'a' => '(a) add a new owner',
            'r' => '(r) rename owner',
            'i' => '(i) ignore owner in this session',
            'q' => '(q) quit and save'
          }
```
and have like `prompt_options` as a method:

```ruby
def prompt_options
  DEFAULT_OPTIONS.merge( suggested_owner ? {'c' => "(c) correct to #{suggested_owner}"})
end
```

Reply:
Great option! :ok_hand: